### DISCUSSION TITLE
60fps+ overhaul, ultra-wide, widescreens, layer edges...

### DISCUSSION THREAD

**Discussion by snouz** [Albeoris/Memoria#522] [Releases]
![image](https://github.com/Albeoris/Memoria/assets/9964576/1f6d88a9-0554-432a-ba06-d7c783c1f90f)
[COMPLETE CHANGELOG HERE](https://github.com/Albeoris/Memoria/wiki/Changelog-v2024.06.09)

## Features
- High Framerate (60/120fps) overhaul
  - Battles -> Position/rotation/scale of camera, actors, shadows, 2D sfx and skies
  - World map -> Position/rotation/animation of camera, moving objects and shadows
  - Fields -> Position/rotation/animation of camera, actors, shadows and objects
- All resolutions (16:10 to ultrawide) can now be widescreen (16:9 still recommended)
- Edges between foreground layers and 3D objects/effects are now clean [#352](https://github.com/Albeoris/Memoria/issues/352) (thanks to @EricHu33)
- Turn-based re-balance: removed the wait when enemies act, closer to vanilla in difficulty
- Worldmap rotation with R-Stick
- +10 backgrounds and +6 scenes are now widescreen

## Fixes
- Many bugfixes (60fps+, UI, battles, smooth cams, widescreens, turbo dialogs, truncated names, Fossil Roo path, sounds, layers, launcher...)
- Launcher: changed some defaults settings

## Modding
- Moddable shaders, gil symbol and DoubleCast

@SamsamTS @snouz @Tirlititi @DV666 @EricHu33

<hr /><em>This discussion was created from the release <a href='https://github.com/Albeoris/Memoria/releases/tag/'>60fps+ overhaul, ultra-wide, widescreens, layer edges...</a>.</em>

**Comment by dennissuarezz** [comment]
There was a big bug with freyas trance jump sound, it is totally fixed in this version thank you!
